config/application.rb

```diff
@@ -22,22 +22,25 @@ class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 5.1 # NOTE: [Rails 6] we should at least work towards updating this to 5.2
 
+    # [Rails 6] Zeitwerk is the new autoloader
+    # As we don't have `load_defaults 6.0` yet, it has to be enabled manually
+    # See <https://guides.rubyonrails.org/autoloading_and_reloading_constants.html>
+    config.autoloader = :zeitwerk
+
+    # Disable auto adding of default load paths to $LOAD_PATH
+    # Setting this to false saves Ruby from checking these directories when
+    # resolving require calls with relative paths, and saves Bootsnap work and
+    # RAM, since it does not need to build an index for them.
+    # see https://github.com/rails/rails/blob/6-0-stable/railties/CHANGELOG.md#rails-600rc2-july-22-2019
+    config.add_autoload_paths_to_load_path = false
+
     # Settings in config/environments/* take precedence over those specified here.
     # Application configuration can go into files in config/initializers
     # -- all .rb files in that directory are automatically loaded after loading
     # the framework and any gems in your application.
 
-    config.autoload_paths += Dir["#{config.root}/app/labor/"]
-    config.autoload_paths += Dir["#{config.root}/app/decorators/"]
-    config.autoload_paths += Dir["#{config.root}/app/services/"]
-    config.autoload_paths += Dir["#{config.root}/app/presenters/"]
-    config.autoload_paths += Dir["#{config.root}/app/liquid_tags/"]
-    config.autoload_paths += Dir["#{config.root}/app/black_box/"]
-    config.autoload_paths += Dir["#{config.root}/app/sanitizers"]
-    config.autoload_paths += Dir["#{config.root}/app/facades"]
-    config.autoload_paths += Dir["#{config.root}/app/errors"]
-    config.autoload_paths += Dir["#{config.root}/app/view_objects"]
```

Comment:
every folder under `/app` is autoloaded by default in zeitwerk mode

Reply:
it's so clean now ✨

Reply:
Rails 5 already autoloaded all direct descendants of `/app`, so this configuration shouldn't have been necessary.